src/main/java/net/coderbot/iris/pipeline/HandRenderer.java

```diff
@@ -64,8 +65,17 @@ public boolean isAnyHandTranslucent() {
 		return isHandTranslucent(InteractionHand.MAIN_HAND) || isHandTranslucent(InteractionHand.OFF_HAND);
 	}
 
+	public Matrix4f getHandProjectionMatrix(GameRenderer gameRenderer, Camera camera, float tickDelta) {
```

Comment:
This isn't really what I meant - I mean that we can take the result of getProjectionMatrix, and construct a new matrix (call it "scale") that just contains the scale call, then call scaleMatrix.multiply(getProjectionMatrix(...)) or something like that.

Reply:
I attempted to do that, but the hand ended up turning invisible due to it not being called before the embedded multiply call of getProjectionMatrix.